systemaudioplatform/systemaudioplatform.h

```diff
@@ -0,0 +1,52 @@
+#ifndef SYSTEMAUDIO_PLATFORM
+#define SYSTEMAUDIO_PLATFORM
+#include <stdint.h>
+#include <string>
+#include <gst/gst.h>
+#include <gst/audio/audio.h>
+
+   enum AudioType
```

Comment:
Please make sure that the alignment is following a standard.
Suggestion is to follow https://www.kernel.org/doc/html/v4.10/process/coding-style.html